### PR TITLE
Easier rebuild of native modules

### PR DESCRIPTION
**Pull request by ffflorian**
* When we rebuild native modules, we can now set the target architecture with the environment variable `wire_target_arch`.
* Updated `electron-rebuild`.

### CODE REVIEW

File: electron/rebuild-native-modules.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+'use strict';
+
+const spawn = require('cross-spawn');
+const exec = require('child_process').exec;
+const arch = process.env.wire_target_arch ? process.env.wire_target_arch : process.arch;
+
+const normalize = (args) => {
+  return args.map((arg) => {
+    Object.keys(process.env).forEach((key) => {
+      const regex = new RegExp(`\\$${key}|%${key}%`, 'i');
```
Comment (bennyn) [inline electron/rebuild-native-modules.js]: More descriptive name for `regex`. What does it match?

File: electron/rebuild-native-modules.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    return arg;
+  });
+};
+
+let args = process.argv.slice(2);
+if (args.length === 1) {
+  const [command] = normalize(args);
+  const proc = exec(command, (error, stdout, stderr) => {
+    if (error) {
+      console.error(`exec error: ${error}`);
```
Comment (bennyn) [inline electron/rebuild-native-modules.js]: It's always good to mention the original error object, like:

```javascript
console.error(`exec error: ${error}`, error);
```

File: electron/rebuild-native-modules.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    return arg;
+  });
+};
+
+let args = process.argv.slice(2);
+if (args.length === 1) {
+  const [command] = normalize(args);
+  const proc = exec(command, (error, stdout, stderr) => {
+    if (error) {
+      console.error('Execution error: ', error);
```
Comment (lipis) [inline electron/rebuild-native-modules.js]: no need for colon space (`: `) when using multi arguments
```
console.error('Execution error', error);
```

Comment (ffflorian) [inline electron/rebuild-native-modules.js]: True, no need for the space - but the colon makes the error more readable.

**Pull request merged**